docs/connectors/sources/http.md

```diff
@@ -1,16 +1,87 @@
-# HTTP Input
+# HTTP input connector
 
-Feldera supports pushing data to a SQL table over HTTP.  For more information,
-see
+Feldera supports directly pushing data to a SQL table over HTTP.
 
-* [Tutorial section](/docs/tutorials/basics/part2) on HTTP-based input and
-  output.
+* This input connector is ephemeral: it is created when the HTTP
+  connection is opened, and is deleted once it is closed.
 
-* [REST API documentation](https://www.feldera.com/api/push-data-to-a-sql-table)
-  for the `/ingress` endpoint.
+* It is the only input connector not created and managed by the user.
 
-:::caution Under Construction
+* Usage is through a special endpoint:
 
-This section is under construction.
+  **[/v0/pipelines/:pipeline_name/ingress/:table_name?format=...](https://www.feldera.com/api/push-data-to-a-sql-table)**
```

Comment:
I haven't see the `:` notation before.  Is it standard in REST API docs to represent variable parts of the URL like that instead of, e.g., `<pipeline_name>`?

Reply:
Yes, that's the standard notation. The swagger docs use that style too.

Reply:
Yes, I kept it like that for now, I usually do the `<...>`, but for consistency for now used that syntax